ERC721A.sol

```diff
@@ -325,6 +327,25 @@ contract ERC721A is
   function _exists(uint256 tokenId) internal view returns (bool) {
     return tokenId < currentIndex;
   }
+  
+  function _safeBurn(uint256 tokenId) internal {
+    _safeBurn(tokenId);
+  }
+
+  /**
+   * @dev Burns a certain token from id.
+   *
+   * Emits a {Transfer} event.
+   */
+  function _safeBurn(
+    uint256 tokenId,
+    bytes memory _data
```

Comment:
is there a reason for this parameter not being used?

Reply:
I thought this was needed for receiver, but feel free to remove if it is not needed.

Reply:
I want to ask for _safeMint function as well

Reply:
I think it should be kept; makes it easier for other implementations to pass/use arbitrary data, should they want to

Reply:
I agree

Reply:
yup, I agree too, thanks for the clarification!